redaxo/src/addons/media_manager/lib/managed_media.php

```diff
@@ -10,12 +10,33 @@ class rex_managed_media
     public const PROP_WEBP_QUALITY = 'webp_quality';
     public const PROP_INTERLACE = 'interlace';
 
+    /**
+     * @var string|null
+     */
     private $mediaPath = '';
+    /**
+     * @var string|null
```

Comment:
ich meine, das darf nicht null sein. 100% sicher bin ich aber nicht

Reply:
Es ist zumindest default null und wird in construct nicht initialisiert.. daher ist es technisch erstmal null

Reply:
doch, über setMediaPath

Reply:
Ok, dann müssen wir den construct param aber auch als non-nullable deklarieren

Reply:
Wäre auch korrekt glaube ich